README.md

```diff
@@ -97,13 +97,14 @@ To tear down all of the resources, run:
 ## Modules
```

Comment:
Could this section and the following "Running examples" section be reordered? I think it might be useful for someone just trying out the tool to have all the example uses (i.e. quickstart and the example configurations) in the same place.